reference/functions/pgroonga-highlight-html.md

```diff
@@ -80,7 +80,25 @@ CREATE INDEX pgroonga_content_index
               normalizer='NormalizerNFKC100');
 ```
 
-Now, you can use `pgroonga_content_index` as `index_name`:
+If you specify `NormalizerTable`, like in this `CREATE INDEX USING pgroonga` document ([`How to use NormalizerTable` section](https://pgroonga.github.io/reference/create-index-using-pgroonga.html#normalizer-table)), the specified PGroonga index must have `NormalizerTable` with `"report_source_offset", true"` option.
+
+Here is an example:
+
+```sql
+CREATE INDEX pgroonga_memos_index
+          ON memos
+       USING pgroonga (content)
+        WITH (normalizers='
+                NormalizerNFKC130,
+                NormalizerTable(
+                  "normalized", "${table:pgrn_normalizations_index}.normalized",
+                  "target", "target",
+                  "report_source_offset", true
+                )
+             ');
+```
+
+Here is an example of using `pgroonga_highlight_html` function:
```

Comment:
Could you add one more example instead of changing existing simple example?
A simple example is still useful to understand what is happen.

Reply:
@kou san

OK. I will make a new simple example for this.